src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(x) {
+    return function getSum1(y) {
+      counter++;
+
+      if (counter >= 4 && (counter % 2) === 0) {
```

Comment:
```suggestion
      if (counter >= 4 && counter % 2 === 0) {
```
Думаю можно и без скобок

Reply:
Yes, it will work.